src/styles/cdr-reset.scss

```diff
@@ -114,9 +114,4 @@ input[type='month'] {
   -webkit-appearance: none;
 }
 
-[data-palette='sandstone'] {
-  --cdr-color-background-surface-primary: #{$cdr-color-background-secondary};
-  --cdr-color-background-surface-secondary: #dcd6cb;
-  --cdr-color-border-surface-primary: #{$cdr-color-border-secondary};
-  --cdr-color-border-surface-secondary: #726d64;
-}
+
```

Comment:
With this removal, how are users expected to add in palettes? Maybe they would need to import the palettes file manually?

Reply:
I think this question still applies—should we export the palette sass file you created?

Reply:
I ended up moving this file to styles root dir and allowing it to be built to dist